source/docs/software/actuators/pneumatics.rst

```diff
@@ -112,3 +112,24 @@ Double solenoids are controlled by the DoubleSolenoid class in WPILib. These are
         exampleDouble.Set(frc::DoubleSolenoid::Value::kOff);
         exampleDouble.Set(frc::DoubleSolenoid::Value::kForward);
         exampleDouble.Set(frc::DoubleSolenoid::Value::kReverse);
+
+
+Pressure Sensors
+----------------
+
+To read the pressure stored in the pneumatic system, connecting an analog pressure sensor is possible. These sensors connect to the Analog ports on the roboRIO, and can be read by the AnalogInput class in WPILib.
```

Comment:
Consider rewording the first sentence to "One can connect a pressure sensor to measure the pressure stored in a pneumatic system."

Reply:
Specify Analog Input port On RoboRIO, since there are also Analog Output ports

Reply:
Fixed both.